benchmarking/commons/utils.py

```diff
@@ -74,7 +74,7 @@ def find_or_create_requirements_txt(
 ) -> Path:
     if requirements_fname is None:
         requirements_fname = "requirements.txt"
-    files = list(entry_point.parent.rglob("requirements*.txt"))
+    files = list(entry_point.parent.glob("requirements*.txt"))
```

Comment:
What was the issue that this fixes?

Reply:
This is supposed to find a file "requirements*.txt" in the current path, but not in subdirectories.

Reply:
In general, I was using `rglob` at several places when I should have used `glob`. This is fixed